README.md

```diff
@@ -11,6 +11,13 @@
 [![Maintainability](https://api.codeclimate.com/v1/badges/fb243c31ea059c0038b2/maintainability)](https://codeclimate.com/repos/608b5886f52398018b00264c/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/fb243c31ea059c0038b2/test_coverage)](https://codeclimate.com/repos/608b5886f52398018b00264c/test_coverage)
 
+[Quick start](#quick-start) • [What you get](#what-you-get) •
+[Commands](#useful-commands) • [Configuration](#configuration) •
+[Roadmap](#developer-roadmap) • [Debugging](#debugging) •
+[Contributing](#contributing) • [Telemetry](#data-and-telemetry)
+
+---
```

Comment:
It now looks like this @shcheklein 

![image](https://user-images.githubusercontent.com/1477535/176307618-95aa3c93-c7aa-4bab-9179-ec3b8bbc93b2.png)

But you can remove the HR if you prefer:

```suggestion
```